### PR TITLE
Add instructions for compiling the documentation

### PR DESCRIPTION
**Pull request by gassmoeller**
Add some instructions for how to compile the documentation.

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -235,8 +235,26 @@ If the configure process fails to create Machine_Definitions, your build is a ch
...
+sphinx (including its dependencies)
+pandoc
+python
+numpy
+scipy
+sphinxcontrib-bibtex
+nbsphinx
+jupyter-client
+ipykernel
+
```
Comment (cianwilson) [inline INSTALL]: The `master` branch currently lists in `doc/requirements.txt` (in addition to those that are here already): recommonmark and matplotlib.  Should they be added to the list?

File: INSTALL [hunk 2]
```diff
@@ -235,8 +235,26 @@ If the configure process fails to create Machine_Definitions, your build is a ch
...
-
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+V Building the documentation
+
+If you plan to modify the documentation you may want to build it locally to
+check your changes before submission. To do so you need a number of additional
+dependencies that are not required to build the Rayleigh executable. In
+particular you need the following packages:
+
+sphinx (including its dependencies)
```
Comment (cianwilson) [inline INSTALL]: What dependencies are being referred to here?  Don't all of the following have dependencies that aren't mentioned?

Comment (feathern) [reply INSTALL]: I agree.   For now, I would remove the parenthical statement.   I have some loose instructions on how to get a conda environment set up to visualize Rayleigh data.   I can update those to include Sphinx etc.   I think simply listing the base packages in the INSTALL file is sufficient for now.

**Comment by gassmoeller** [comment]
Thanks for merging. You were right Cian, the instructions here were somewhat outdated (the documentation setup changed too quickly to keep them updated).

**Pull request merged**